packages/grid/_modules_/grid/components/column-header-item.tsx

```diff
@@ -62,35 +76,40 @@ export const ColumnHeaderItem = React.memo(
           width,
           minWidth: width,
           maxWidth: width,
-          maxHeight: headerHeight,
-          minHeight: headerHeight,
         }}
         role="columnheader"
         tabIndex={-1}
         aria-colindex={colIndex + 1}
         {...ariaSort}
       >
-        {column.type === 'number' && (
-          <ColumnHeaderSortIcon
-            direction={column.sortDirection}
-            index={column.sortIndex}
-            hide={column.hideSortIcons}
-          />
-        )}
-        {headerComponent || (
-          <ColumnHeaderTitle
-            label={column.headerName || column.field}
-            description={column.description}
-            columnWidth={width}
-          />
-        )}
-        {column.type !== 'number' && (
-          <ColumnHeaderSortIcon
-            direction={column.sortDirection}
-            index={column.sortIndex}
-            hide={column.hideSortIcons}
-          />
-        )}
+        <div
+          className="MuiDataGrid-colCell-draggable"
```

Comment:
However, we could start using this approach, it can be clearer. I think that it can work fine, especially for the part of the Grid that we don't allow developers to customize (public components with the `components` prop). For the parts that we want to make public, we could have a `DataGridColCell` component and a class name:

```suggestion
          className="MuiDataGridColCell-draggable"
```

If we do move in this direction. I think that it should require to update all the other occurrences. We could use the name of the component as a secondary separator and rename `packages/grid/_modules_/grid/components/column-header-item.tsx` to `packages/grid/_modules_/grid/components/GridColCell.tsx` to match the class name.

Happy to open a new issue for this.

Reply:
Let's create a separate ticket for this because I guess we would need to rename other files as well.

Reply:
Ok done #408